webpack/assets/javascripts/react_app/components/common/BindingComponent.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
```

Comment:
When using ES6, it is necessary to bind functions to 'this'

The binding component just makes it easier